### PR TITLE
[fix](nereids)DOUBLE acos (DOUBLE x) returns the inverse cosine value of x…

### PR DESCRIPTION
**Pull request by penglele**
…, or nan if x is not in the range of -1 to 1

## Proposed changes

Issue Number: close #xxx

Execute in the nereids optimizer：
mysql> select acos(-2);
ERROR 1105 (HY000): NumberFormatException, msg: null

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by morningman — COMMENTED** [review]
Please add test case

File: fe/fe-core/src/main/java/org/apache/doris/common/util/LiteralUtils.java [hunk 1]
```diff
@@ -32,7 +32,9 @@
 public class LiteralUtils {
 
     public static String getStringValue(FloatLiteral literal) {
-        if (literal.getType() == Type.TIME || literal.getType() == Type.TIMEV2) {
+        if (Double.isNaN(literal.getValue())) {
+            return "nan";
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/LiteralUtils.java:36]: we should not return nan. and this is not a right way to fix this problem. in sql, we could only return valid value of the type of this column or Null literal.

**Review by morrySnow — CHANGES_REQUESTED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/common/util/LiteralUtils.java (hunk 1 above)
Comment (penglele) [reply fe/fe-core/src/main/java/org/apache/doris/common/util/LiteralUtils.java:36]: if (Double.isNaN(literal.getValue())) {
            return Double.toString(literal.getValue());
        }
so changed?

Comment (penglele) [reply fe/fe-core/src/main/java/org/apache/doris/common/util/LiteralUtils.java]: @morrySnow Do I change it as above, or do you change it?

**Pull request closed without merging**